ntbs-service-unit-tests/Helpers/SeedingHelperTest.cs

```diff
@@ -8,17 +8,18 @@ public class SeedingHelperTest
     {
         [Fact]
         public void GetRecordsFromCSV_GetsTBServicesData() {
-            var results = SeedingHelper.GetRecordsFromCSV<TBService>("../../../TestData/tbservices.csv");
+            var results = SeedingHelper.GetTBServices("../../../TestData/tbservices.csv");
             var firstEntryInTBServicesList = new TBService {
                 Code = "TBS0001",
-                Name = "Abingdon Community Hospital"
+                Name = "Abingdon Community Hospital",
+                PHECCode = "E45000019"
             };
             const int expectedCount = 6;
 
             Assert.Equal(results.Count, expectedCount);
-            Assert.Equal(results[0].Code, firstEntryInTBServicesList.Code);
-            Assert.Equal(results[0].Name, firstEntryInTBServicesList.Name);
-
+            Assert.Equal(((TBService)results[0]).Code, firstEntryInTBServicesList.Code);

```

Comment:
Very minor point here, but convention is `expected, actual`

Reply:
good spot!